Projects/UOContent/Spells/Base/SpellHelper.cs

```diff
@@ -220,7 +220,7 @@ public static bool CheckCombat(Mobile m)
                 }
             }
 
-            if (Core.Expansion == Expansion.AOS)
+            if (Core.AOS)
```

Comment:
I think this was supposed to specifically be for only AOS and not later?
Let's double check if possible what the different eras should be.

Reply:
ServUO doesnt check it for expansion at all.